frontend/app_flowy/assets/translations/pt-BR.json

```diff
@@ -0,0 +1,146 @@
+{
+    "appName": "AppFlowy",
+    "defaultUsername": "Me",
+    "welcomeText": "Bem vindo @:appName",
+    "githubStarText": "Star on GitHub",
+    "subscribeNewsletterText": "Se inscreva na Newsletter",
+    "letsGoButtonText": "Vamos lá",
+    "title": "Título",
+    "signUp": {
+      "buttonText": "Inscreve-se",
+      "title": "Inscrever-se @:appName",
+      "getStartedText": "Começar",
+      "emptyPasswordError": "Senha não pode ser em branco.",
+      "repeatPasswordEmptyError": "Confirmar a senha não pode ser em branco.",
+      "unmatchedPasswordError": "As senhas não conferem.",
+      "alreadyHaveAnAccount": "Já possui uma conta?",
+      "emailHint": "Email",
+      "passwordHint": "Senha",
+      "repeatPasswordHint": "Confirme a senha"
+    },
+    "signIn": {
+      "loginTitle": "Login to @:appName",
+      "loginButtonText": "Login",
+      "buttonText": "Entre",
+      "forgotPassword": "Esqueceu a senha?",
+      "emailHint": "Email",
+      "passwordHint": "Senha",
+      "dontHaveAnAccount": "Não possui uma conta?",
+      "repeatPasswordEmptyError": "Confirmar a senha não pode ser em branco.",
+      "unmatchedPasswordError": "As senhas não conferem."
+    },
+    "workspace": {
+      "create": "Crie uma área de trabalho",
+      "hint": "área de trabalho",
+      "notFoundError": "Área de trabalho não encontrada"
+    },
+    "shareAction": {
+      "buttonText": "Compartilhar",
+      "workInProgress": "Em breve",
+      "markdown": "Markdown",
+      "copyLink": "Copiar o link"
+    },
+    "disclosureAction": {
+      "rename": "Renomear",
+      "delete": "Deletar",
+      "duplicate": "Duplicar"
+    },
+    "blankPageTitle": "Página em branco",
+    "newPageText": "Nova página",
+    "trash": {
+      "text": "Lixeira",
+      "restoreAll": "Restaurar todos",
+      "deleteAll": "Deletar todos",
+      "pageHeader": {
+        "fileName": "Nome do arquivo",
+        "lastModified": "Última modificação",
+        "created": "Criado"
+      }
+    },
+    "deletePagePrompt": {
+      "text": "Está página está na lixeira",
+      "restore": "Restaurar a oágina",
```

Comment:
página

Reply:
Has this been resolved? @lays147 @carnei-ro